plugins/modules/eos_l3_interfaces.py

```diff
@@ -41,7 +41,7 @@
 notes:
 - Tested against Arista EOS 4.20.10M
 - This module works with connection C(network_cli). See the L(EOS Platform Options,../network/user_guide/platform_eos.html).
-  'eos_l2_interfaces/eos_interfaces' should be used for preparing the node , before applying L3 configurations using
+  'eos_l2_interfaces/eos_interfaces' should be used for preparing the interfaces , before applying L3 configurations using
```

Comment:
This doesn't make sense to me, because based on your code in #105 that you have proposed, you are proposing to _not_ use `eos_l2_interfaces` to set the port mode to either L2 or L3, so why are you saying that we need all three now?